Resources/Prototypes/Entities/Objects/Tools/pka_upgrade.yml

```
@@ -54,10 +55,14 @@
     - state: overlay-3
       color: "#1373eb"
   - type: GunUpgrade
-    tags: [ GunUpgradeRange ]
+    #tags: [ GunUpgradeRange ] # DeltaV - allow multiple
     examineText: gun-upgrade-examine-text-range
   - type: GunUpgradeSpeed
     coefficient: 1.5
```

```suggestion
    coefficient: 1.4 # Harmony - 1.5 > 1.4 - Prevents clipping at max stacks
```
Tested, found this to be the maximum viable value